pocs/observatory.py

```diff
@@ -342,67 +341,24 @@ def update_tracking(self):
-                dec_direction = 'south'
+            # Get the pier side of pointing image
+            pointing_ha = self.current_observation.pointing_image.header_ha
 
-            # find the number of ms and direction for RA axis
-            ra_offset = self.current_offset_info.delta_ra
-            ra_ms = self.mount.get_ms_offset(ra_offset, axis='ra')
-            if ra_offset >= 0:
-                ra_direction = 'west'
-            else:
-                ra_direction = 'east'
-
-            dec_ms = abs(dec_ms.value) * 1.5  # TODO(wtgee): Figure out why 1.5
-            ra_ms = abs(ra_ms.value) * 1.
-
-            # Ensure we don't try to move for too long
-            max_time = 99999
-
-            # Correct the Dec axis (if offset is large enough)
-            if dec_ms > max_time:
-                dec_ms = max_time
-
-            if dec_ms >= 50:
-                self.logger.info("Adjusting Dec: {} {:0.2f} ms {:0.2f}".format(
-                    dec_direction, dec_ms, dec_offset))
-                if dec_ms >= 1. and dec_ms <= max_time:
-                    self.mount.query('move_ms_{}'.format(
-                        dec_direction), '{:05.0f}'.format(dec_ms))
-
-                # Adjust tracking for up to 30 seconds then fail if not done.
-                start_tracking_time = current_time()
-                while self.mount.is_tracking is False:
-                    if (current_time() - start_tracking_time).sec > 30:
-                        raise Exception("Trying to adjust Dec tracking for more than 30 seconds")
-
-                    self.logger.debug("Waiting for Dec tracking adjustment")
-                    time.sleep(0.1)
-
-            # Correct the RA axis (if offset is large enough)
-            if ra_ms > max_time:
-                ra_ms = max_time
-
-            if ra_ms >= 50:
-                self.logger.info("Adjusting RA: {} {:0.2f} ms {:0.2f}".format(
-                    ra_direction, ra_ms, ra_offset))
-                if ra_ms >= 1. and ra_ms <= max_time:
-                    self.mount.query('move_ms_{}'.format(
-                        ra_direction), '{:05.0f}'.format(ra_ms))
-
-                # Adjust tracking for up to 30 seconds then fail if not done.
-                start_tracking_time = current_time()
-                while self.mount.is_tracking is False:
-                    if (current_time() - start_tracking_time).sec > 30:
-                        raise Exception("Trying to adjust RA tracking for more than 30 seconds")
-
-                    self.logger.debug("Waiting for RA tracking adjustment")
-                    time.sleep(0.1)
+            try:
+                pointing_ha = pointing_ha.value
+            except AttributeError:
+                pass
+
+            self.logger.debug("Pointing HA: {}".format(pointing_ha))
+            correction_info = self.mount.get_tracking_correction(
+                self.current_offset_info,
+                pointing_ha
+            )
+
+            try:
```

Comment:
Is it worth putting in a limit, where we don't make a correction if the error is sufficiently low?

Reply:
It happens back in [`get_tracking_correction`](https://github.com/panoptes/POCS/blob/8d6d6aa0926ef4362964eec22530691698e13516/pocs/mount/mount.py#L366) where it checks for anything under 50 ms. Nothing goes into the returned dict so the `correct_tracking` method then just skips that whole axis. 

The 50 ms was determined empirically by me where I noticed when it was shorter than about 40ms it would just happen too quickly and then the system would get stuck waiting. It's also less than 1 pix of movement.

I'll make a note in the docstring to `get_tracking_correction` about how the values are clipped for min max.